### DISCUSSION TITLE
Option for mono/stereo sounds

### DISCUSSION THREAD

**Discussion by tizfra** [GeekFunkLabs/fluidpatcher#42] [Q&A] [answered]
Is it possible to play squishbox in mono mode to connect it to a single speaker?
I think is possible to add L-R balance in effect, or something in options...

**Comment by albedozero** [comment, marked as answer]
Sure - if you only plug a cable into the Right/Mono jack it will receive a combined Left/Right signal. You can also control balance and left/right panning with control changes 8 and 10, respectively. Balance is supposedly for stereo samples and pan for mono, but they both sound very similar in practice.

![SquishBox diagram](https://geekfunklabs.com/wp-content/uploads/2022/04/squishbox_diagram.png)
